adam_core/propagator/propagator.py

```diff
@@ -12,26 +15,71 @@
 from ..time import Timestamp
 from .utils import _iterate_chunks
 
+os.environ["OMP_NUM_THREADS"] = "1"
```

Comment:
what's all this about?

Reply:
I noticed each ray worker was using 213 threads so this was an attempt to prevent that. Didn't work and I'll remove it before this gets merged.